lib/sanitize.rb

```diff
@@ -77,7 +77,7 @@ class << Sanitize
   def initialize(config = {})
     @config = Config.merge(Config::DEFAULT, config)
 
-    @transformers = Array(@config[:transformers].dup)
+    @transformers = Array(@config[:transformers])
```

Comment:
The reason for the `.dup` call is that we're about to modify the `@transformers` array, and we need to make sure we don't modify an array we don't own. `Array(thing_thats_already_an_array)` returns the original array, not a copy.

This should be fine though:

```ruby
@transformers = Array(@config[:transformers]).dup
```